examples/direct-upload/README.md

```diff
@@ -421,8 +423,8 @@ server.register(require('inert'),
     path: '/{filename*}',
     handler: {
       directory: {
-        path: __dirname + '/public',
-        listing: false,
+        path: 'public',
```

Comment:
does this _work_ without path resolution? (_guessing it does... didn't know..._)